app/scripts/modules/google/instance/details/instance.details.controller.js

```diff
@@ -215,7 +217,17 @@ module.exports = angular.module('spinnaker.instance.detail.gce.controller', [
 
     this.canDeregisterFromLoadBalancer = function() {
       var instance = $scope.instance;
-      if (!instance.loadBalancers || !instance.loadBalancers.length) {
+      // TODO(dpeach): remove when it's possible to degister from l7 load balancer.
```

Comment:
Instance deregister won't be available until we can add individual instances to MIGs. So not in the near future.